internal/service/ssmcontacts/ssmcontacts_test.go

```diff
@@ -28,12 +28,14 @@ func TestAccSSMContacts_serial(t *testing.T) {
 			"tags":          testAccSSMContactsContactDataSource_tagsSerial,
 		},
 		"ContactChannelResource": {
-			acctest.CtBasic:      testAccContactChannel_basic,
-			"contactId":          testAccContactChannel_contactID,
-			"deliveryAddress":    testAccContactChannel_deliveryAddress,
-			acctest.CtDisappears: testAccContactChannel_disappears,
-			acctest.CtName:       testAccContactChannel_name,
-			"type":               testAccContactChannel_type,
+			acctest.CtBasic:             testAccContactChannel_basic,
+			"contactId":                 testAccContactChannel_contactID,
+			"deliveryAddress":           testAccContactChannel_deliveryAddress,
+			acctest.CtDisappears:        testAccContactChannel_disappears,
+			acctest.CtName:              testAccContactChannel_name,
+			"type":                      testAccContactChannel_type,
+			"identity_Basic":            testAccSSMContactsContactChannel_Identity_Basic,
+			"identity_ExistingResource": testAccSSMContactsContactChannel_Identity_ExistingResource,
```

Comment:
Lets remove the `_` here for consistency with other test names.

```suggestion
			"identityBasic":            testAccSSMContactsContactChannel_Identity_Basic,
			"identityExistingResource": testAccSSMContactsContactChannel_Identity_ExistingResource,
```